src/components/EstimatedFee.tsx

```diff
@@ -13,33 +13,39 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
-import { Typography } from '@material-ui/core';
+
 import React from 'react';
-import { useEffect } from 'react';
-import { useState } from 'react';
+import { Typography, makeStyles } from '@material-ui/core';
 import { useSourceTarget } from '../contexts/SourceTargetContextProvider';
 import { useTransactionContext } from '../contexts/TransactionContext';
-import { transformToBaseUnit } from '../util/evalUnits';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    minHeight: '20px'
+  }
+}));
 
 export const EstimatedFee = (): React.ReactElement => {
+  const classes = useStyles();
   const { sourceChainDetails, targetChainDetails } = useSourceTarget();
-  const { estimatedFee, estimatedFeeLoading } = useTransactionContext();
-  const srcChainDecimals = sourceChainDetails.apiConnection.api.registry.chainDecimals[0];
+  const { estimatedFeeLoading, estimatedFeeTransformed } = useTransactionContext();
   const { chainTokens } = targetChainDetails.apiConnection.api.registry;
 
-  const [amount, setAmount] = useState<string>('0');
-
-  useEffect(() => {
-    !estimatedFeeLoading && setAmount(transformToBaseUnit(estimatedFee || '0', srcChainDecimals));
-  }, [estimatedFee, estimatedFeeLoading, srcChainDecimals]);
+  if (!estimatedFeeTransformed) {
+    return <div className={classes.container}></div>;
+  }
 
   return (
-    <div>
+    <div className={classes.container}>
       <Typography variant="body1" color="secondary">
-        {`Estimated ${sourceChainDetails.chain} fee ${estimatedFeeLoading ? '...' : ':'}`}
-        <Typography variant="subtitle2" component="span">
-          {amount} {chainTokens}
-        </Typography>
+        Estimated {sourceChainDetails.chain} fee:{' '}
```

Comment:
Im not sure why you revert [a proposal from Sveta](https://github.com/paritytech/parity-bridges-ui/pull/218#discussion_r671310165).

Reply:
My guess was that what @goldsteinsveta meant is:
```
"Estimated fee..." # when loading
```
and 
```
"Estimated fee: 5 RLT" # when ready
```

This one is a bit closer than previous, but we still have the extra double colon when loading. Also let's have Sveta confirm.

Reply:
> My guess was that what @goldsteinsveta meant is:
> 
> ```
> "Estimated fee..." # when loading
> ```
> 
> and
> 
> ```
> "Estimated fee: 5 RLT" # when ready
> ```
> 
> This one is a bit closer than previous, but we still have the extra double colon when loading. Also let's have Sveta confirm.

I changed to work in this way.